src/Data/Rational/Unnormalised/Properties.agda

```diff
@@ -1670,9 +1678,27 @@ neg-distrib-⊓-⊔ = antimono-≤-distrib-⊓ neg-mono-≤
   ℤ.+ n ℤ.* 1ℤ       ∎)
   where open ℤ.≤-Reasoning
 
-∣p∣≡p∨∣p∣≡-p : ∀ p → (∣ p ∣ ≡ p) ⊎ (∣ p ∣ ≡ - p)
-∣p∣≡p∨∣p∣≡-p (mkℚᵘ (ℤ.+ n)    d-1) = inj₁ refl
-∣p∣≡p∨∣p∣≡-p (mkℚᵘ (-[1+ n ]) d-1) = inj₂ refl
+∣p∣≡p⊎∣p∣≡-p : ∀ p → (∣ p ∣ ≡ p) ⊎ (∣ p ∣ ≡ - p)
+∣p∣≡p⊎∣p∣≡-p (mkℚᵘ (ℤ.+ n)    d-1) = inj₁ refl
+∣p∣≡p⊎∣p∣≡-p (mkℚᵘ (-[1+ n ]) d-1) = inj₂ refl
+
+
+∣p∣≃p⇒0≤p : ∀ {p} → ∣ p ∣ ≃ p → 0ℚᵘ ≤ p
+∣p∣≃p⇒0≤p {p} ∣p∣≃p with ∣p∣≡p⊎∣p∣≡-p p
+∣p∣≃p⇒0≤p {p} ∣p∣≃p | inj₁ ∣p∣≡p  = ∣p∣≡p⇒0≤p ∣p∣≡p
```

Comment:
In the two cases you can omit everything to the left of the `|` symbol by replacing it with `...`

Reply:
Fixed